src/common/listItem/ListItem.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
I'm not sure if we should have a separate directory for ListItem component. Why not just keep it grouped with the List component under the same directory?